static/src/javascripts/projects/commercial/modules/user-features.spec.js

```diff
@@ -250,53 +257,50 @@ describe('Storing new feature data', () => {
                 adFree: true,
             })
         );
-        refresh().then(() => {
+        return refresh().then(() => {
             expect(getCookie(PERSISTENCE_KEYS.PAYING_MEMBER_COOKIE)).toBe(
                 'true'
             );
-            expect(getCookie(PERSISTENCE_KEYS.AD_FREE_USER_COOKIE)).toBe(
-                'true'
-            );
+            expect(
+                getCookie(PERSISTENCE_KEYS.AD_FREE_USER_COOKIE)
+            ).toBeTruthy();
```

Comment:
The value it has is a string version of a timestamp, e.g. `"1499271595715"` so this assertion was actually failing before (but the test itself didn't fail because it wasn't set up for async).

We test that it has the correct format in the other tests, so I just left it as simple as possible as the point of this one is just to prove that it has been populated.